mautic/mautic
# Fixed Portugal regions names in regions.json

Fixed Portugal regions names in regions.json

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Yes
| New feature? | No 
| Automated tests included? | No
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | No
| Deprecations? | No

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
The portuguese regions naming had a few mistakes.
[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Go to create a new contact
2. Text in State dropdown

#### Steps to test this PR:

- Checkout the staging branch
- Add some contacts with regions in Portugal, e.g. Braganca (you can see the full list in https://github.com/mautic/mautic/pull/8134/files)
- Checkout this PR
- Run `bin/console doctrine:migrations:migrate`
- The contacts will now have the updated values (without the DB migration you'd see an empty state field)

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
